keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#include "ergodox.h"
+// #include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+#include "eeconfig.h"
+#include "eeprom.h"
+#include "keymap_colemak.h"
```

Comment:
If you use `SEND_STRING`, and use software colemak (eg, in your OS), then you may want to consider including "sendstring_colemak.h" as well.